src/item/VanillaItems.php

```diff
@@ -545,16 +549,17 @@ public function isFireProof() : bool{ return true; }
 
 		foreach(BoatType::getAll() as $type){
 			//boat type is static, because different types of wood may have different properties
-			self::register($type->name() . "_boat", new Boat(new IID(match($type){
+			self::register($type->name() . "_" . (mb_strtolower($type->getSuffix() ?? "boat", 'US-ASCII')), new Boat(new IID(match($type){
 				BoatType::OAK() => Ids::OAK_BOAT,
 				BoatType::SPRUCE() => Ids::SPRUCE_BOAT,
 				BoatType::BIRCH() => Ids::BIRCH_BOAT,
 				BoatType::JUNGLE() => Ids::JUNGLE_BOAT,
 				BoatType::ACACIA() => Ids::ACACIA_BOAT,
 				BoatType::DARK_OAK() => Ids::DARK_OAK_BOAT,
 				BoatType::MANGROVE() => Ids::MANGROVE_BOAT,
+				BoatType::BAMBOO() => Ids::BAMBOO_RAFT,
```

Comment:
I'm not really sure this is appropriate. It also wasn't mentioned in the PR description.

Reply:
The name of the PR mentions bamboo items and therefore indirectly bamboo boats, which are above all boats.

Reply:
I'm not convinced that it's appropriate to consider rafts as boats. They have different behaviour and appearance.

Reply:
You can suffocate on a raft when going under 1 block, but not on a boat, since you sit higher on a raft than on a boat.